src/reynir_correct/pattern.py

```diff
@@ -1707,6 +1707,90 @@ def né(self, match: SimpleTree) -> None:
             )
         )
 
+    def agreement_conj(self, match: SimpleTree) -> None:
+        vp = match.first_match("VP > (so_ft|so_et)")
+        if vp is None:
+            return
+        so = vp.first_match("so")
+        if so is None:
+            return
+        start, end = so.span
+        sbj = match.first_match("NP-SUBJ")
+        if sbj is None:
+            return
+        if len(sbj) > 1:  # TODO: more accurate subject selection
+            sbj = sbj[0]
+        variants = [f for f in so.all_variants if f != "vh"]
+        variants.append("fh")
+        suggest = self.get_wordform(so.lemma, so.cat, variants)
+        if not suggest:
+            return
+        text = f"Hér á sögnin '{so.lemma}' að samræmast frumlaginu '{sbj.lemma}'"
+        self._ann.append(
+            Annotation(
+                start=start,
+                end=end,
+                code="P_NT",
+                text=text,
+                original=so.tidy_text,
+                suggest=suggest,
+            )
+        )
+
+    def agreement_subpost_sing(self, match: SimpleTree) -> None:
```

Comment:
No pattern in the script triggers this function. Should I delete it or keep it in case a pattern will be added?